examples/go.mod

```diff
@@ -1,16 +1,19 @@
 module acme.corp
 
-go 1.22.0
+go 1.23.0
 
 replace acme.corp/pkg => ./pkg
 
+replace github.com/kcp-dev/client-go => ./rebased_kcp/client-go
```

Comment:
Dependency on client-go, which in turn depends on code-generator. Do we update `examples/` in a separate PR?

Reply:
(also see #75, btw)